Classes/Domain/Repository/JobRepository.php

```diff
@@ -61,14 +61,14 @@ public function findOneRunning(): ?Job
             ->where(
                 $queryBuilder->expr()->eq(
                     'status',
-                    $queryBuilder->createNamedParameter(Job::STATUS_RUNNING, \PDO::PARAM_INT)
+                    $queryBuilder->createNamedParameter(Job::STATUS_RUNNING, Connection::PARAM_INT)
                 )
             )
             ->orderBy('created_time', 'ASC')
             ->setMaxResults(1)
-            ->execute()
-            ->fetch();
-        if ($row === false) {
+            ->executeQuery()
+            ->fetchAllAssociative();
+        if ($row === []) {
```

Comment:
`fetchAssociative()` "Returns the next row of the result as an associative array or FALSE if there are no more rows."
i think the old condition was right.

Reply:
ah, sorry, here you use fetchAllAssociative(), so conditiion is correct, but above it is wrong.

but here the fetchAllAssociative() seems the wrong method, you should use fetchAssociative(), to get a single row, because
`(new Job())->fromDatabaseRow($row)` expect on database row, and not an array of database rows
i think you have to recheck the fetch-Methods

Reply:
Thanks @achimfritz fixed it.